frontend/app/src/components/helper/HashLink.vue

```diff
@@ -105,9 +114,12 @@ const { href, onLinkClick } = useLinks(url);
 
 <template>
   <div class="d-flex flex-row shrink align-center">
-    <v-avatar v-if="showIcon" size="24" class="mr-2">
-      <v-img :src="makeBlockie(displayText)" />
-    </v-avatar>
+    <!--    TODO: move makeBlockie to local, and make the size adjustable-->
+    <!--    <span>-->
+    <!--      <v-avatar v-if="showIcon && !tx" size="24" class="ml-n2 mr-2">-->
+    <!--        <v-img :src="makeBlockie(displayText)" />-->
+    <!--      </v-avatar>-->
+    <!--    </span>-->
```

Comment:
A bit more info about this? 

Btw since we talked about the performance of base64. I think in the past in another project I was working on we did a small LRU cache so the last 30-40 blockies where actually stored in memory instead of being re-calculated every time. This certainly improved performance.

Maybe we can do the same here.

Reply:
Yeah exactly what I was thinking, store them in memory. But I will do it in another PR, since I think it little bit complicated, and has nothing to do with this PR.